scripts/launcher/lib/virtual_controller.py

```diff
@@ -132,10 +134,9 @@ def _prepare_args(self):
             args.append("--network")
             args.append(nic)
 
-        if self._iso.stage2:
-            disk_opts = "path={0},device=cdrom,readonly=on,shareable=on".format(self._iso.iso_path)
-            args.append("--disk")
-            args.append(disk_opts)
+        disk_opts = "path={0},device=cdrom,readonly=on,shareable=on".format(self._iso)
+        args.append("--disk")
+        args.append(disk_opts)
 
         if self._ks_paths:
             extra_args = "ks=file:/{0}".format(os.path.basename(self._ks_paths[0]))
```

Comment:
I think this should be rather `inst.ks=...` since we're working towards making the prefix mandatory.

Reply:
Dito.